definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -5272,26 +5307,131 @@
     "CRUD": "R"
   },
   {
-    "LiveFrom": "18/07/2019",
+    "LiveFrom": "25/06/2019",
     "CaseTypeID": "DIVORCE",
-    "CaseEventID": "solicitorSubmitAOS",
-    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CaseEventID": "IssueAOSOffline_CoRespondent_fromAosStarted",
+    "UserRole": "caseworker-divorce-superuser",
     "CRUD": "R"
   },
   {
     "LiveFrom": "25/06/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "IssueAOSOffline_CoRespondent_fromAosStarted",
-    "UserRole": "caseworker-divorce-superuser",
+    "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "R"
   },
   {
-    "LiveFrom": "25/06/2019",
+    "LiveFrom": "07/07/2020",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "IssueAOSOffline_CoRespondent_fromAosStarted",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "07/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_Respondent_fromAosAwaiting",
+    "UserRole": "caseworker-divorce-courtadmin",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "07/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_Respondent_fromAosAwaiting",
```

Comment:
@jvictorcf in the content of the ticket (DIV-5559) you've created is says we should support that event. But it doesn't exist.